src/components/disclosure/components/disclosure-activator/index.tsx

```diff
@@ -0,0 +1,37 @@
+// Third-party imports
+import classNames from 'classnames'
+
+// Global imports
+import { useDisclosure } from 'components/disclosure/provider'
+import disclosureStyles from 'components/disclosure/disclosure.module.css'
+
+// Local imports
+import { DisclosureActivatorProps } from './types'
+
+/**
+ * Component for rendering the always-visible activator within a `Disclosure`.
+ *
+ * @see https://developer.hashi-mktg.com/swingset/components/disclosure
+ */
+const DisclosureActivator = ({
```

Comment:
**Thought 💭** With the `Accordion` component noted as coming soon, curious if we should use [`forwardRef`](https://reactjs.org/docs/forwarding-refs.html) with the `DisclosureActivator` so that when the time comes to programmatically set focus to the activators via keyboard we can do via refs?

Reply:
Ooooh great call! I'm going to think on this for a few 🤔

Reply:
🗒️ Added in https://github.com/hashicorp/dev-portal/pull/481/commits/d2537567fdfebad3c66dc4192a4769a34282e78e